src/Resources/translations/CmfSonataAdminIntegrationBundle.de.xliff

```diff
@@ -287,8 +284,7 @@
             <!-- $Core -->
             <trans-unit id="form.tab_publish">
                 <source>form.tab_publish</source>
-                <!-- TODO: Translate -->
-                <target>Publish Settings</target>
+                <target>Einstellungen zum Veröffentlichen</target>
```

Comment:
My german is not that great, but in the other publish workflow related messages "publizieren" was used instead of "veröffentlichen". Can this be made more consistent? (e.g. can we use "publizieren" here as well or is that incorrect?)